$NetBSD: patch-aa,v 1.1 2008/11/08 12:55:21 bjs Exp $

Code could rely on the definition of 'VERSION'--and *not* define
it in installed headers!

If this patch causes a problem, something is not right.

--- include/mutils/mhash_config.h.in.orig	2007-04-04 22:21:23.000000000 -0400
+++ include/mutils/mhash_config.h.in
@@ -224,7 +224,7 @@
 #undef USE_DMALLOC
 
 /* Version number of package */
-#undef VERSION
+/* #undef VERSION */
 
 /* Define if using the dmalloc debugging malloc package */
 #undef WITH_DMALLOC
